frontend/redux/sagas.js

```diff
@@ -36,11 +38,22 @@ export function* requestWorkload() {
   }
 }
 
+export function* completeWorkload() {
+  try {
+    const workloadToSend = yield select(selectWorkload);
+    const workloadPostResponse = yield call(postWorkload, workloadToSend);
+    yield put(workloadPostResponse);
```

Comment:
I feel like `workloadPostResponse` needs to be passed into an action creator to dispatch the "WORKLOAD_RECEIVED" or whatever